docs/meetings/2021/TechArea20210720.md

```diff
@@ -42,16 +41,26 @@
     -   AI (Carl): Missing zero walltime records
     -   AI (BrianL): Use vomsxrd config in osg-xrootd on 3.6
     -   AI (TimT): SciTokens credmon?
+-   AI (BrianL): Give GlideinWMS team JIRA access.
 
 
 ### Discussion
 
-None this week  
+-   GlideinWMS 3.7 release candidate planned for tomorrow (7/21).
+
+-   We are currently only sending LIGO jobs to Georgia Tech to avoid having OSG
+    (and other VO) jobs run on LIGO pilots.
 
 
 ### Support Update
 
-None this week  
+-   Marian:
```

Comment:
```suggestion
-   BrianL:
    -   We are currently only sending LIGO jobs to Georgia Tech to avoid having OSG
        (and other VO) jobs run on LIGO pilots.
-   Marian:
```